src/main/java/teammates/storage/entity/StudentProfile.java

```diff
@@ -18,6 +19,11 @@
 @PersistenceCapable
 public class StudentProfile {
 
+    // CHECKSTYLE.OFF:JavadocVariable self-documenting
+    @NotPersistent
+    public static final String PRIMARY_KEY_NAME = Entity.getFieldWithPrimaryKeyAnnotation(StudentProfile.class);
```

Comment:
I guess you're right. Let's instead add a proper documentation? Something like
```java
/**
 * The name of the primary key of this entity type.
 */
```
Redundant as it may, it looks better than suppressing CheckStyle.